src/sass/components/_intro-modal.scss

```diff
@@ -434,6 +434,10 @@
   -webkit-overflow-scrolling: auto;
   overflow-x: hidden;
 
+  &.advisers {
```

Comment:
Hi @josh-southern, could you take a look at the BEM naming conventions, linked to from here? https://github.com/wevote/WebApp/blob/develop/docs/working/STYLING.md

I would recommend naming this &__advisers (instead of &.advisers) so that the class you would use would be: intro-modal-vertical-scroll-contain__advisers (instead of "advisers")